### PR TITLE
gale: init at 1.1happy

### PR DESCRIPTION
**Pull request by drvink**
Gale messaging system

### CODE REVIEW

File: nixos/modules/services/networking/gale.nix [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  # config gets rebuilt if the public key changes (we can assume the private key
+  # will never change without the public key having changed)
+  gpubFile = cfg.keyPath + "/${cfg.domain}.gpub";
+  home = "/var/lib/gale";
+  keysPrepared = cfg.keyPath != null && lib.pathExists cfg.keyPath;
+in
+{
+  options = {
+    services.gale = {
+      enable = mkOption {
```
Comment (jagajaga) [inline nixos/modules/services/networking/gale.nix]: Please use `mkEnableOption`

File: pkgs/applications/networking/instant-messengers/gale/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  preConfigure = ''
+    substituteInPlace configure.ac --replace \$\{sysconfdir\} /etc
+    ./bootstrap
+  '';
+  configureArgs = [ "--sysconfdir=/etc" ];
+
+  meta = with stdenv.lib; {
+    homepage = "http://gale.org/";
+    description = "Gale messaging system (server and client)";
```
Comment (jagajaga) [inline pkgs/applications/networking/instant-messengers/gale/default.nix]: Please omit pkg's name in the description.

**Comment by drvink** [comment]
Fixed the issues you mentioned and pushed an updated commit.

**Comment by lethalman** [comment]
Travis passed, thanks.

**Pull request merged**